lib/util/cloud_messaging_handler.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:firebase_messaging/firebase_messaging.dart';
+
+class CloudMessanger {
```

Comment:
Rename `CloudMessanger` to something like `CloudMessaging` or `MessagingHandler` or `FCMHandler`